scripts/DESeq_functions.R

```diff
@@ -270,9 +270,9 @@ annotate_deseq_table <- function(deseq_results_list, params, filter_results = F,
       ## FILTERS ##
       if (biosets_filter == T) {
         # for biosets, filter on unadjusted p-value
-        deg_table <- deg_table[!is.na(deg_table$pval) & deg_table$pval < params$alpha & abs(deg_table$linearFoldChange) > params$linear_fc_filter, ]
+        deg_table <- deg_table[!is.na(deg_table$pval) & deg_table$pval < params$alpha & abs(deg_table$linearFoldChange) > 1.2, ]
       }else if (filter_results == T) {
-        deg_table <- deg_table[!is.na(deg_table$padj) & deg_table$padj < params$alpha & abs(deg_table$linearFoldChange) > params$linear_fc_filter, ]
+        deg_table <- deg_table[!is.na(deg_table$padj) & deg_table$padj < params$alpha & abs(deg_table$linearFoldChange) > 1.2, ]
```

Comment:
Conversely... doesn't this block get used for non-biosets? Because it's filtering on $padj instead of $pval. Am I correct in thinking that this one should be left alone?

Reply:
I really do not know. Maybe we could talk tomorrow about where this block gets used? I can't figure out where biosets_filter == T/F and filter_results == T/F come from.

Having both lines does not change the DEG output files, at least. I checked that.

Reply:
Oh the parameters are in the function. So it uses either no filter, or the bioset filter, or the Deg filter. When we call annotate deseq table we can get these different outputs. Happy to take a look tomorrow together and we can confirm that it's working the way I think.

It may be that the degs in the test data don't change simply because there are none output between 1.2 and 1.5 in the test data...